go-controller/README.md

```diff
@@ -12,10 +12,12 @@ make clean
 make
 ```
 
-Then find the executable here : go-controller/_output/go/bin/ovnkube
+Then find the executables here : go-controller/_output/go/bin/
 
 'ovnkube' is the single all-in-one executable useful for deploying the ovn cni plugin for a kubernetes deployment
 
+'ovn-k8s-cni-overlay' is the cni executable to be planted under /opt/cni/bin so that it can be invoked for each pod event
```

Comment:
This is moved to /usr/libexec

Reply:
Only on Fedora or other distros that don't like /opt though.  It'll still be /opt/cni/bin for others.

Reply:
Question: how do I build it for the fedora RPM?